### PR TITLE
chore: add more unittest for `MsgIssue` of x/token

### PR DESCRIPTION
**Pull request by zemyblue**
## Description

* modify the input validate unittest of x/token MsgIssue 
* Add test case of MsgIssue to check conditions and actions

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/token/keeper/msg_server_test.go [hunk 1]
```diff
@@ -264,6 +265,106 @@ func (s *KeeperTestSuite) TestMsgIssue() {
 	}
 }
 
+func (s *KeeperTestSuite) TestMsgIssue2() {
```
Comment (tkxkd0159) [inline x/token/keeper/msg_server_test.go:268]: Why don't you integrate this tests with previous one?

Comment (zemyblue) [reply x/token/keeper/msg_server_test.go:268]: Oh, I did not remove previous one to reference when I develop. I'll remove it. Thank you.

**Review by jaeseung-bae — COMMENTED** [review]
How about this indentation for events?
스크린샷 2023-09-21 오후 5 34 09

**Comment by zemyblue** [comment]
> How about this indentation for events?
> 스크린샷 2023-09-21 오후 5 34 09

That's great. But don't you think is it too long to understand total codes?

**Comment by jaeseung-bae** [comment]
> > How about this indentation for events?
> > 스크린샷 2023-09-21 오후 5 34 09
> 
> That's great. But don't you think is it too long to understand total codes?

I think so, it is vertically too long. But the events in the original one are horizontally too long.
Let's keep it as is.

**Review by jaeseung-bae — APPROVED** [review]

**Review by tkxkd0159 — APPROVED** [review]

**Pull request merged**